bundler.js

```diff
@@ -91,7 +91,16 @@ export async function installBundler(bundlerVersionInput, lockFile, platform, ru
   } else {
     const gem = path.join(rubyPrefix, 'bin', 'gem')
     const bundlerVersionConstraint = /^\d+\.\d+\.\d+/.test(bundlerVersion) ? bundlerVersion : `~> ${bundlerVersion}`
-    await exec.exec(gem, ['install', 'bundler', '-v', bundlerVersionConstraint])
+    const floatVersion = common.floatVersion(rubyVersion)
+    if ((bundlerVersionConstraint === '~> 2') && (floatVersion >= 2.3) && (floatVersion <= 2.5)) {
+      // stop warnings similar to
+      // Your RubyGems version (2.6.14.4)) has a bug that prevents `required_ruby_version` from working for Bundler
+      await exec.exec(gem, ['update', '--system', '--no-document'])
+    } else if (platform.startsWith('windows-') && (floatVersion === 3.1)) {
+      await exec.exec(gem, ['install', 'bundler', '-v', bundlerVersionConstraint, '--force'])
+    } else {
+      await exec.exec(gem, ['install', 'bundler', '-v', bundlerVersionConstraint])
+    }
```

Comment:
Could you drop the changes in this file and rebase to master?

Reply:
done